### PR TITLE
Data types references

### PR DESCRIPTION
**Pull request by sofietoft**
- Add a section about Data Types in the Tracking article.
- Added a link to the Tracking article from the Data Types article.

### CODE REVIEW

**Review by eshanrnh — CHANGES_REQUESTED** [review]
Looks good to me 💪 Just one comment - In the Data types article, we are already mentioning the steps on how to view the data type references so I don't think we should link to the tracking article for the same information.

File: Fundamentals/Data/Data-Types/index.md [hunk 1]
```diff
@@ -44,6 +44,9 @@ To view the Data Type reference, go to the __Settings__ section and expand the _
 ![Content Picker References](images/viewing-data-type-reference.png)
 
 This gives you an overview of the Types that currently use the Data Type.
+
+Learn more about tracking Data Type references in the [Tracking](../../../Extending/Property-Editors/Tracking/) article.
```
[the remark this answers is not in the document]
Comment (sofietoft) [reply Fundamentals/Data/Data-Types/index.md:48]: True! I was just thinking that if someone wanted to learn more about tracking... 
There are also some code samples in the tracking article, that might be relevant!

We can leave it out, for sure!

[the remark this answers is not in the document]
Comment (eshanrnh) [reply Fundamentals/Data/Data-Types/index.md:48]: Ah ok, then maybe you can rephrase it as:

```suggestion
Learn more about viewing references or implementing tracking in the [Tracking References](../../../Extending/Property-Editors/Tracking/) article.
```

**Comment by sofietoft** [comment]
Thanks for the suggestions 💪 
I've committed the changes. Will merge it in!

**Pull request merged**